libraries/config/src/config.cpp

```diff
@@ -270,6 +270,10 @@ void NetworkConfig::validate() const {
                           std::to_string(MAX_PACKETS_PROCESSING_THREADS_NUM) + "]");
   }
 
+  if (network_transaction_interval < 0) {
```

Comment:
The type of network_transaction_interval is uint16_t, that will not < 0

Reply:
true, the condition is supposed to be <= 0, thanks